src/dns/aardvark.rs

```diff
@@ -144,6 +146,36 @@ impl Aardvark {
         Ok(())
     }
     pub fn commit_entries(&self, entries: Vec<AardvarkEntry>) -> Result<()> {
+        // Acquire fs lock to ensure other instance of aardvark cannot commit
+        // or start aardvark instance till already running instance has not
+        // completed its `commit` phase.
+        let lockfile_path = Path::new(&self.config)
```

Comment:
Path for lock file is intentionally `<run-root-base>/networks/aardvark.lock` instead of `<run-root-base>/networks/aardvark-dns/aardvark.lock` to make this patch backward compatible with older `aardvark` versions.

Reply:
cc @Luap99 @mheon just fyi we will have a lock file now for aardvark commits, but we can change path later if there is any disagreement on this.